app/js/components/orderEntry/Accordion.jsx

```diff
@@ -26,7 +26,7 @@ class Accordion extends React.Component {
                 <i className="toggle-icon icon-caret-right small rotate90" />
               )}
             </span>
-            {format(date, dateFormat)}
+            {format(date, dateFormat).slice(0, -5)}
```

Comment:
hey, why slicing through the dates, the `date format`is a global property that is set in every standalone. so this might look well on your standalone but not good in another's. so I'd suggest you remove the slicing and come up with a general dateformat every user should set their standalone to.

Reply:
The standard date format is DD-MM-YYYY HH:mm. Therefore, I removed the Time in this case.